ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/NoActiveUserManagementView.kt

```diff
@@ -56,31 +66,53 @@ private fun ContentUnavailableView(
     description: String?,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(ContentUnavailableViewPadding),
-        horizontalAlignment = Alignment.CenterHorizontally,
+    Surface(
+        modifier = modifier,
+        shape = RoundedCornerShape(CustomerCenterConstants.Card.ROUNDED_CORNER_SIZE),
+        color = MaterialTheme.colorScheme.surface,
     ) {
-        Icon(
-            imageVector = Icons.Rounded.Warning,
-            contentDescription = null,
-            modifier = Modifier.size(ContentUnavailableIconSize),
-        )
-
-        Text(
-            text = title,
-            style = MaterialTheme.typography.headlineSmall,
-            modifier = Modifier.padding(top = ContentUnavailableViewPaddingTopTitle),
-        )
+        Column(
+            modifier = Modifier.fillMaxWidth()
+                .padding(all = 20.dp),
```

Comment:
Hmm I wonder if the horizontal padding should be smaller... But I'm ok if we think it looks good!